### PR TITLE
Spike sassdocs integration

### PR DESCRIPTION
**Pull request by nickcolley**
There's a fair bit of complexity in the render view logic which would be nice to simplify but this is hopefully a good proof of concept for us to talk about.

### CODE REVIEW

File: source/sass_api_reference/index.html.md.erb [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        <% end %>
+
+        <% if doc.context.type == 'variable' %>
+```scss
+$<%= doc.type.downcase %>: $<%= doc.context.name %>;
+```
+        <% end %>
+
+        <% if doc.example %>
+#### Example
```
Comment (nickcolley) [inline source/sass_api_reference/index.html.md.erb]: I don't like how the markdown needs to be indented differently to the rest of the file for it to render properly, makes it hard to read...

**Comment by m-green** [comment]
So good to see this in the TDT! Awesome work Nick.
Quick question - is is possible to remove the `<hr>` at the end of each subsection?

**Comment by nickcolley** [comment]
@m-green done

**Comment by nickcolley** [comment]
Source code links should be permalinked to the specific version being deployed

**Comment by nickcolley** [comment]
If we updated tech docs gem to allow `disable_indented_code_blocks: true` we could indent the file and have it read out easily.

**Comment by nickcolley** [comment]
Added a change to handle deprecation warnings:

**Comment by nickcolley** [comment]
https://github.com/alphagov/tech-docs-gem/pull/170

File: source/sass_api_reference/index.html.md.erb (hunk 1 above)
Comment (nickcolley) [reply source/sass_api_reference/index.html.md.erb]: https://github.com/alphagov/tech-docs-gem/pull/170

**Pull request closed without merging**